servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/RunnableCompletable.java

```diff
@@ -17,15 +17,11 @@
 
 import io.servicetalk.concurrent.internal.ThreadInterruptingCancellable;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import static io.servicetalk.concurrent.internal.SubscriberUtils.handleExceptionFromOnSubscribe;
+import static io.servicetalk.concurrent.internal.SubscriberUtils.safeOnComplete;
 import static java.lang.Thread.currentThread;
 import static java.util.Objects.requireNonNull;
 
-public class RunnableCompletable extends AbstractSynchronousCompletable {
-    private static final Logger LOGGER = LoggerFactory.getLogger(RunnableCompletable.class);
+final class RunnableCompletable extends AbstractSynchronousCompletable {
```

Comment:
this is an API breaking change, but I think this was an unintentional omission. can you confirm @blake-bauman?

Reply:
https://github.com/apple/servicetalk/pull/1686

Reply:
Correct.  This must've been an oversight from the review.  I didn't make this public intentionally.